bundler/lib/bundler/templates/Executable.standalone

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env <%= Bundler.settings[:shebang] || RbConfig::CONFIG["ruby_install_name"] %>
+# frozen_string_literal: true
```

Comment:
This one would be a different cop, right? I guess could be proposed separately.

Reply:
This also came up as an offense now that the template is correctly covered by RuboCop:

```
bundler/lib/bundler/templates/Executable.standalone:1:1: C: [Correctable] Style/FrozenStringLiteralComment: Missing frozen string literal comment.
#!/usr/bin/env <%= Bundler.settings[:shebang] || RbConfig::CONFIG["ruby_install_name"] %>
^
```

I could split this in two pull-requests, though: one that fixes the RuboCop coverage of the template files. And another one, based on the first one, that introduces the performance cop. What do you think?

Reply:
I think two pull requests is a good idea yeah. Or at least separate commits.

Reply:
I went for two separate commits, then: one for correctly calling the cops on the binstubs (plus the minor documentation change), and one that introduces `Performance/StringInclude` and its fixes.